docs/types/literal-suffixes.rst

```diff
@@ -0,0 +1,243 @@
+|                                                             | - :ref:`String literals<string_literals>`                      |
+|                                                             | - :ref:`Unicode literals<unicode_literals>`                    |
+|                                                             | - :ref:`Zero<rational_literals>`                               |
++-------------------------------------------------------------+----------------------------------------------------------------+
+| ``bytes``                                                   | - :ref:`Hexadecimal string literals<hexadecimal_literals>`     |
+|                                                             | - :ref:`String literals<string_literals>`                      |
+|                                                             | - :ref:`Unicode literals<unicode_literals>`                    |
++-------------------------------------------------------------+----------------------------------------------------------------+
+| ``string``                                                  | - :ref:`String literals<string_literals>`                      |
+|                                                             | - :ref:`Unicode literals<unicode_literals>`                    |
++-------------------------------------------------------------+----------------------------------------------------------------+
+
+For two-parameter suffix functions, the first parameter (representing the mantissa) can be of any integer type.
+The second parameter (the exponent) must be of an unsigned integer type.
+
+.. note::
+    40-digit literals prefixed with ``0x`` such as, for example, ``0xdCad3a6d3569DF655070DEd06cb7A1b2Ccd1D3AF``
+    always represent ``address`` literals in the language.
+    To invoke a suffix accepting ``bytes20`` you must use one of the other literal kinds implicitly
+    convertible to ``bytes20``, e.g., a hexadecimal string literal
+    (``hex"dCad3a6d3569DF655070DEd06cb7A1b2Ccd1D3AF"``).
+
+.. note::
+    Suffix functions accepting ``address payable`` are not allowed since address literals are never payable.
+
+Suffix functions may not accept or return reference types with ``storage`` or ``calldata`` locations.
+
+.. index:: function;call
+.. _calling_suffix_functions:
+
+Calling Suffix Functions
+------------------------
+
+There are two ways to call a suffix function:
+
+#. Suffix call.
+#. Function call.
+
+.. index:: ! literal suffix; suffix call syntax
+
+Suffix Call Syntax
+^^^^^^^^^^^^^^^^^^
+
+A suffix call has the same syntax as a literal with a :ref:`denomination<denominations>`:
+
+.. code-block:: solidity
+
+    42 suffix;
+    1.23 suffix;
+    "abc" suffix;
+    hex"12ff"suffix;
+    true suffix;
+
+The literal passed as input to the suffix function must be immediately followed by the name of the suffix.
+The two must be separated by whitespace unless it's a string, unicode or hexadecimal string literal,
```

Comment:
maybe also add the unicode in the list examples above?

Reply:
Added.